.travis.yml

```diff
@@ -90,3 +90,6 @@ after_failure:
 notifications:
     email:
       - dev@tomcat.apache.org
+cache:
+  directories:
+  - $HOME/.m2
```

Comment:
Tomcat does not use Maven so I see no reason to cache this folder.

Reply:
I add this because I see this repo contains pom.xml....

Reply:
Where is it ?
Did you confuse it with `build.xml` (Ant) ?

Reply:
Or maybe the ones in https://github.com/apache/tomcat/blob/main/modules/ ?
I am not sure whether Ant executes those. So the cache might be useful after all.